src/components/PageSidebar/SidebarTabSharing.vue

```diff
@@ -49,16 +100,72 @@ export default {
 		},
 	},
 
+	data() {
+		return {
+			membersManagementHelpText: t('collectives', 'Share this page with members of your collective. Members can view and edit the shared content based on their permissions.'),
```

Comment:
```suggestion
			membersManagementHelpText: t('collectives', 'Manage members of your collective. Members have access to the whole collective.'),
```

There's no way to add members for a specific page, so I'd rephrase it slightly to make obvious that members have access to the full collective. Also I think we should not use the term "sharing" for managing members, as it's something conceptionally different, but I'm unsure here. Would be curious what you think @silverkszlo.